### ISSUE TITLE
Is Oscillator S&H Noise or SH Noise?

### ISSUE THREAD

**Issue by esaruoho** [surge-synthesizer/surge#431]
seems like the manual says S&H noise and the dropdownmenu says SH Noise. Which is it, @kurasu ?

**Comment by baconpaul** [comment]
My bet - having not tried it - is that the windows convention of using &X as an accelerator in a menu means that "S&H Noise" is a bad menu item name on windows. 

https://support.microsoft.com/en-us/help/261320/how-to-use-an-ampersand-in-a-form-name-menu-or-control

You could change it to "S+H Noise" if you really want. The string is in resources/data/configuration.xml.

**Comment by sense-amr** [comment]
i dont beleive S&HNOISE will fit in the allotted space

**Comment by sense-amr** [comment]
ahh yes that makes sense .. S+H Noise is probably the best then .. but i was refering to the space of the bitmap not the menu itself .. that is the area where the bitmap is called from the menu option it does seem somewhat space constrained?

**Comment by sense-amr** [comment]
So that was another reason i was suggested he opted for SHNOISE as he did for W-TABLE cos theres just no space there..

**Comment by sense-amr** [comment]
CLASSIC and SHNOISE are the same amount of characters.. as is W-TABLE

**Comment by baconpaul** [comment]
Yeah I don't think we should change the bitmap!

**Comment by sense-amr** [comment]
i do think SHNOISE seems descriptive enough to know what it means .. it could be changed just in the context menu to be the same then that would be more uniform .. or both the bitmap and the context menu could be changed to just S+H which is what it really is anyway.. the "Noise" bit is a tad redundant..

**Comment by baconpaul** [comment]
Yeah I think @esaruoho makes a choice and updates the manual! easy enough :)

**Comment by baconpaul** [comment]
@esaruoho to quote a great Canadian band, "if you choose not to decide you still have made a choice" :)

What would you like to do here?

**Comment by mkruselj** [comment]
If you want to show & on Windows, you escape it with another ampersand. So it should be S&&H Noise. :)

**Comment by baconpaul** [comment]
Man, you sound like baconpaul on jan31 ;)
Unfortunately that shows a double ampersand on mac and windows as I mentioned then!

**Comment by mkruselj** [comment]
Ah, sorry! But shouldn't it be possible to condition a double ampersand if Windows OS is detected (via ifdef)?

**Comment by mkruselj** [comment]
Ah nvm then. I read the issue all the way through now. :)
